ea_restaurant_data_loader/lib/auth/token.ex

```diff
@@ -0,0 +1,3 @@
+defmodule EaRestaurantDataLoader.Token do
```

Comment:
why not to directly use `Joken.Config` and eliminate this file?

Reply:
This is the way how library recommend the use of Joken.Config, This way it will implement some functions for you.

https://hexdocs.pm/joken/Joken.Config.html

Reply:
Fair enough!